README.md

```diff
@@ -1,18 +1,60 @@
 # canvas-lti-redirect-tool
 This is Canvas LTI redirect tool which use the CAI developed [LTI library](https://pypi.org/project/django-lti/)
 
-## Generating Django security key
+### Prerequisites
```

Comment:
Consider the readme, as of my notes. But I am going to make this clean in future